### PR TITLE
Project Deployment using Github Pages

### PR DESCRIPTION
**Pull request by cancelei**
Hello Reviewer, at this stage of the project I deploy the website using Github Pages.

**Highlights:**
- Github Pages Hosted.
- Minor Readme Update to make it more accessible.
- I also included the link on the repo itself (right-side bar), so recruiters can look it up when it's ready.

Happy review!

### CODE REVIEW

File: README.md
Comment (skyv26) [inline README.md:1]: - [ ] Wonderful 🚀 , you did very good work and deployed the project successfully using `gh-pages` but I noticed that your `README` not looks professional. I saw some of your `Getting Started` sub-sections don't have the correct information or you used `N/A` in *install*, *Usage* and *Deployment*, which is actually not a good idea and not useful for that person who wants to understand how your project is working. Please I would `humbly` suggest you add valuable information, you are almost done after adding mentioned issues. Kindly have a look at below comparison table to understand the problem very well.

<table>
<tr>
<th>Must Look Like</th>
<th>Your README content</th>
</tr>
<tr>
<td>

Screenshot from 2023-05-08 18-55-11

</td>
<td>

image

</td>
</tr>
</table>

**Note:** *Please just add the relevant information at the place where you added N/A.*

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @cancelei, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Correct `GitHub` flow ✅ 
- Linters are green ✅ 
- Project is Live ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @cancelei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (cancelei) [reply README.md:1]: Thank you @skyv26 ! That was a very complete feedback, I wasn't sure if that section was for that information. Thanks for the details.

Comment (skyv26) [reply README.md:1]: @cancelei I hope and would strongly suggest you add every section except FAQ in your `README` files. Good Luck 👍🏻

**Pull request merged**